charts/incubator/miniflux/CHANGELOG.md

```diff
@@ -2,6 +2,282 @@
 
 
 
+## [miniflux-12.0.0](https://github.com/truecharts/charts/compare/miniflux-11.0.0...miniflux-12.0.0) (2023-11-27)
+
+### Docs
+
+- Add docs
+
+### Feat
+
+- Update charts to latest common and use cnpg
+  - Use upstream container and bump version to 2.0.50
+
+### Style
+
```

Comment:
There was no changelog since multiple version and git-chglog was unable to fetch them due to changes of path (stable to incubator), so I manually added them, we can discord those changes if it's not suitable.

Reply:
This file is not allowed to be edited in any way, shape or form.